ksmt-test/src/test/kotlin/org/ksmt/test/benchmarks/SymFPUBenchmarksBasedTest.kt

```diff
@@ -0,0 +1,142 @@
+package org.ksmt.test.benchmarks
+
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Timeout
+import org.junit.jupiter.api.parallel.Execution
+import org.junit.jupiter.api.parallel.ExecutionMode
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import org.ksmt.KContext
+import org.ksmt.solver.KSolver
+import org.ksmt.solver.bitwuzla.KBitwuzlaSolver
+import org.ksmt.solver.bitwuzla.KBitwuzlaSolverConfiguration
+import org.ksmt.solver.yices.KYicesSolver
+import org.ksmt.solver.yices.KYicesSolverConfiguration
+import org.ksmt.solver.z3.KZ3SMTLibParser
+import org.ksmt.solver.z3.KZ3Solver
+import org.ksmt.solver.z3.KZ3SolverConfiguration
+import org.ksmt.symfpu.solver.SymfpuSolver
+import java.nio.file.Path
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.TimeUnit
+import kotlin.io.path.Path
+import kotlin.io.path.writeLines
+import kotlin.system.measureNanoTime
+import kotlin.time.Duration.Companion.seconds
+
+@Execution(ExecutionMode.SAME_THREAD)
+@Timeout(10, unit = TimeUnit.SECONDS)
+class SymFPUBenchmarksBasedTest : BenchmarksBasedTest() {
```

Comment:
This is a benchmarking infrastructure but not a benchmarks based tests. Let's remove it or rename. Also, let's add benchmarks based tests like we have for all other solvers.